docs/isan_studies___tutorials/uc_davies.md

```diff
@@ -14,7 +14,7 @@ To explore the service and the model, please proceed as follows:
 
     ![CC_inputfiles](https://user-images.githubusercontent.com/32800795/61584726-1d111100-ab4d-11e9-9a91-93c1da82b05c.JPG ':size=500%')
 
-3. Next, click “Retrieve” at the bottom of the worbench and then “Run.” You will then be able to view results of the simulation in any of the viewer services.
+3. Next, click “Run” at the bottom of the worbench. You will then be able to view results of the simulation in any of the viewer services.
```

Comment:
typo: wor**K**bench

Perhaps a spell-checker in vscode can help detecting these typos on the fly
https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
could also add an auto-check either in pre-commit or as a git action for all manuals

Reply:
my concern with the git action is that we have a lot of "wrong" words like "osparc", "neuro-", filenames, code snippets etc. I installed the spell-checker you suggested